src/apps/create-patron-user-info/CreatePatron.dev.tsx

```diff
@@ -134,7 +134,7 @@ export default {
         "The phone number must be 6 to 15 characters in length and should be comprised solely of numbers or begin with a +",
       control: { type: "text" }
     },
-    storybookContextCpr: {
+    fakeCpr: {
```

Comment:
Sorry, but my idea was just to call it: `cpr` :) I can accept `fakeCpr` but I think `cpr` makes more sense.

Reply:
It can't really be called that, as I explained previously there already is a variable called `cpr`.